### PR TITLE
Add test for walltime set to zero after pbs_mom_restart

### PR DESCRIPTION
**Pull request by vstumpf**
#### Bug/feature Description
* Automated test for PP-1117, merged from #521 
* *If there is an issue ID, link to it: [PP-1117](https://pbspro.atlassian.net/browse/PP-1117)*

#### Solution Description
* The filename was changed to `pbs_mom_walltime`, as only one test in the suite deals with hooks.
* Added a new test `test_mom_restart`, that submits a job and waits for the walltime to be more than 30 seconds. It then stops the MoM with a `SIGINT`, restarts it with `-p`, and waits until walltime changes. If it is less than 30, then we've proved the buggy behavior. If it stays above 30, then it works correctly.

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [x] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

[master-momwalltime-20180405111613.txt](https://github.com/PBSPro/pbspro/files/1882156/master-momwalltime-20180405111613.txt)

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_mom_walltime.py [hunk 1]
```diff
@@ -137,3 +137,29 @@ def test_suspend_time_not_counted_in_walltime(self):
         # Used walltime should be less than the sleep time after suspend
         self.server.expect(JOB, {'resources_used.walltime': 10}, op=LE,
                            id=jid1, extend='x')
+
+    def test_mom_restart(self):
+        """
+        Test that time spent on jobs running on MoM will not reset when
+        MoM is restarted
+        """
+
```
Comment (minghui-liu) [inline test/tests/functional/pbs_mom_walltime.py]: I think a blank line here isn't necessary but it's not a big deal.

Comment (vstumpf) [reply test/tests/functional/pbs_mom_walltime.py]: fixed

**Comment by bhroam** [comment]
Looks good.  Please squash and rebase for the merge.

**Comment by bhroam** [comment]
I merged in the PR.

**Comment by vstumpf** [comment]
Thanks @bhroam!
